buildSrc/build.gradle

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
One more: please add build directories and .gradle to .gitignore.

![image](https://user-images.githubusercontent.com/410275/109150839-4cdcbd80-7761-11eb-9924-f9a27269daca.png)

Reply:
Fixed.